### ISSUE TITLE
Could allow single-click to open folders in List View. 

### ISSUE THREAD

**Issue by zhenlxyx** [files-community/Files#7043]
### What's the Problem?

The List View feature looks like the macOS Finder feature in which the user is able to expand the folder hierarchy from left to right. The only difference is that in macOS the "double-click to open folders" logic is changed to "single-click to open folders" in this specific view.

### Solution/Idea

Allow single-click to open folders in List View. 

### Alternatives

If the concern is to keep consistent on folder-opening behavior, a new toggle in App Settings, "Use single-click to open folders in List View" can be added and is on by default. 

### Priorities

| Capability | Priority |
| :---------- | :------- |
| This proposal will allow end users to use single-click to open folders in List View. | Could |

### Files Version

v2.0.34.0

### Windows Version

Windows 11 21H2 22000.318

**Comment by yaichenbaum** [comment]
@zhenlxyx this should already be supported, did you try the option in the files and folders settings page?

**Comment by VOOM108** [comment]
I think this was about being able to set the single click behavior only for the columns view. There is an option for single click behavior generally, but not specifically for columns view. 

Also, there are, in settings, two settings for single click which I cannot discern in German language, I had to switch to English to understand them. Then I realized that they are also labelled wrongly:  

EN
1. Open files with single click
2. Open folders with single click

DE 
1. Öffnen durch einfachen Klick (translating as "Open with single click" - should say "Dateien mit einfachem Klick öffnen")
2. Dateien mit einfachem Klick öffnen (translating as "Open files with single click" - should say "Ordner mit einfachem Klick öffnen")
